src/lsdb/core/crossmatch/abstract_crossmatch_algorithm.py

```diff
@@ -0,0 +1,59 @@
+from abc import ABC, abstractmethod
+from typing import Tuple
+
+import hipscat as hc
+
+import pandas as pd
+
+
+class AbstractCrossmatchAlgorithm(ABC):
+    """Abstract class used to write a crossmatch algorithm"""
+
+    # pylint: disable=too-many-instance-attributes, too-many-arguments
```

Comment:
Please add frequently-annoying pylint configs to https://github.com/lincc-frameworks/python-project-template/issues/256

Reply:
Thanks for pointing that out, I'll look for it in future, but I think these two warnings for arguments and attributes you've already highlighted